### PR TITLE
virgil 5.0.1 (new formula)

### PR DESCRIPTION
**Pull request by hogaco**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by equal-l2** [comment]
When submitting a new formula, append `(new formula)` to the title, for ease of review by Homebrew maintainers.

**Comment by SMillerDev** [comment]
Please stop making a new pull request for each revision of the formula. Just push a new commit to the branch instead.

**Comment by golubovich** [comment]
@SMillerDev , take a look, please :)

**Comment by golubovich** [comment]
ping %)

**Comment by golubovich** [comment]
@lembacon could you take a look, please?

File: Formula/virgil.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Virgil < Formula
+  desc "CLI tool to manage your Virgil account and applications"
+  homepage "https://github.com/VirgilSecurity/virgil-cli"
+  url "https://github.com/VirgilSecurity/virgil-cli.git",
+     :tag      => "v5.0.1",
+     :revision => "03ddee65e9cecbdd7ef55b1285c0ca70758d6d40"
+  head "https://github.com/VirgilSecurity/virgil-cli.git"
+  depends_on "dep" => :build
```
Comment (SMillerDev) [inline Formula/virgil.rb]: Please leave some whitespace here for readability.
```suggestion

    depends_on "dep" => :build
```

File: Formula/virgil.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  depends_on "go" => :build
+  def install
```
Comment (SMillerDev) [inline Formula/virgil.rb]: Same here

**Comment by golubovich** [comment]
@SMillerDev , fixed )

**Comment by golubovich** [comment]
@lembacon, @SMillerDev could you take a look, please?

**Comment by golubovich** [comment]
ping %)

**Comment by golubovich** [comment]
@lembacon, @SMillerDev Hello :) 
I've noticed that more than a month has passed but my formula isn't merged yet. There hasn't been any new comments in the review for 13 days. Is there any other things I can do to get it merged?

**Comment by golubovich** [comment]
ping )

File: Formula/virgil.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    dir.install buildpath.children - [buildpath/".brew_home"]
+    cd dir do
+      system "dep", "ensure", "-vendor-only"
+      system "go", "build", "-o", "virgil"
+      bin.install "virgil"
+    end
+  end
+
+  test do
+    result = `virgil pure keygen`
```
Comment (SMillerDev) [inline Formula/virgil.rb]: This is not a good way to call the function. Please use `shell_output()` like the other formulae.

File: Formula/virgil.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    assert_true result.start_with?("SK.1.")
```
Comment (SMillerDev) [inline Formula/virgil.rb]: Please use `assert_match` here like the other formulae

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @golubovich!

**Pull request closed without merging**